setup.py

```diff
@@ -31,7 +31,6 @@
     install_requires=[
         'click==7.0',
```

Comment:
I believe this is the `click` pinning that matters, the one affecting `pip install shub`.

Reply:
`click` cannot be `>=8.0` for now because Python 2.7 is not supported.

Reply:
Still, can it be higher? e.g. `click >=7, <8`?

Reply:
This was already changed to unbound `click` after skipping the failed tests.